react/components/Link.tsx

```diff
@@ -74,8 +74,12 @@ const Link: React.FunctionComponent<Props> = ({
     return '#'
   }
 
+  const href = getHref()
+  // Href inside admin iframe should omit the `/app/` path
+  const hrefWithoutIframePrefix = domain && domain === 'admin' && href.startsWith('/admin/app/') ? href.replace('/admin/app/', '/admin/') : href
```

Comment:
maybe `domain.toLowerCase()` just to be sure `Admin` won't interfere?

Reply:
That's no need, this domain has just two options (`admin` and `store`) and is not set by the user/developer, but by render.